FoundMarker[] markers
uint8 count
